### ISSUE TITLE
Logo Design

### ISSUE THREAD

**Issue by mmilutinovic013** [elmsln/openulmus.org#26]
@michael-collins has indicated that he would be interested in designing a logo for Open Ulmus. I'm adding an issue here and relating it back to #1

**Comment by michael-collins** [comment]
@cleverington Is there a mission statement somewhere, or something that I can read to understand the brand we are trying to build? Are there existing brands that we are similar to? This would help me get started.

**Comment by michael-collins** [comment]
**Mission statement:** An open collective sustained project for the public production and sharing of knowledge.

**Comment by btopro** [comment]
open collective part less important; more important is the public production and dissemination of knowledge. think if an organization were to have as it's mission to produce HAX, OERSchema and then use ELMS:LN to deploy open materials. We're focusing on FOSS projects leveraging these things but really it's about making anyone into their own Kahn Academy w/ better tools.

**Comment by gusaus** [comment]
I think a big part of the murkiness is we've yet to get input and consensus from the entire team https://openulmus.org/#_team on the high level stuff like mission, goals, plan to sustain, etc.

Unity around https://github.com/openulmus/openulmus.org/issues/57 (and the issues referenced) should provide whoever is tasked with owning #1 to get all those details into the site spec. 

At least that's my take on the situation....

**Comment by cleverington** [comment]
I would disagree on the murkiness, as the Mission statement, in current form, was 'checked off' during our December meeting. Opening discussion on _changing_ the Motto, however, may be needed based on the 501(c3) topic we'll be discussing in our 02/20/2018 team meeting.

The working-doc for Notes / Thoughts on Open Ulmus is here: https://docs.google.com/document/d/11Wz3Qj1IVZHkbDzi0mi4eee4r-ulkh3Qs6pmHB5RIoY/edit 

**Note:** Much of the doc may / may not be out of date, as it does not have a 'lock' functionality, per-say, and kept getting edited.  For the 'state' of the content / verbiage at year's end, just refer to the website and #34.
